src/Optimization/CMakeLists.txt

```diff
@@ -40,10 +42,6 @@ set(hiopOptimization_INTERFACE_HEADERS
   hiopResidual.hpp
   )
 
-if(HIOP_USE_RAJA)
-  list(APPEND hiopOptimization_SPARSE_SRC hiopKKTLinSysSparseCondensed.cpp hiopKKTLinSysSparseNormalEqn.cpp)
-endif()
```

Comment:
This change will lead to `error: 'hiopVectorRajaPar' does not name a type`, e.g. in line 175 of src/Optimization/hiopKKTLinSysSparseCondensed.cpp when building with Ginkgo and CUDA but without RAJA.

Reply:
Thank you! I was having a similar concern re: building w/o RAJA. Nai-Yuan and I will meet today and will address this.

Reply:
I think the issue has been fixed, it would be great if you could confirm